proxies-es6/demo.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+// Proxying a normal object
+var target = {};
+var handler = {
+  get(receiver, name = 'world') {
```

Comment:
I'm not sure what is the value of adding a default value to the `name` parameter there.
How can we even trigger it?